draft-ietf-dprive-dnsoquic.md

```diff
@@ -742,6 +751,11 @@ mitigate these risks. Using session tickets only once mitigates
 the risk of tracking by third parties. Refusing to resume session if addresses
 change mitigates the risk of tracking by the server.
 
+It is unclear whether the tracking attacks also apply in the recursive resolver to
+authoritative server scenario or in the zone transfer scenarios. They might,
+especially if the clients chose to not always initiate connections from the
+same IP address. In that case, the mitigations described above are directly applicable.
+
```

Comment:
How about giving some specific examples here instead:
```
The privacy trade-offs here may be context specific. Stub resolvers will have a strong motivation to prefer privacy over latency since they often change location. However, recursive resolvers that use a static IP address are more likely to prefer the reduced latency provided by session resumption and may consider this a valid reason to re-use resumption tickets more than once. Encrypted zone transfer (RFC9103) explicitly does not attempt to hide the identity of the parties involved in the transfer, but at the same time such transfers are not particularly latency sensitive. 
```

Reply:
I am really not sure about that. There is a fairly strong recommendation in TLS 1.3 to use session tickets just once, so I would be reluctant to give advice that reusing tickets is OK. Also, not all recursive resolvers operate from static IP addresses.

But then, I agree that we should separate recursive and XFR.

Reply:
@saradickinson Please check the new text.